app/less/outfit.less

```diff
@@ -160,3 +160,72 @@ table.total {
     });
   }
 }
+
+#componentPriority {
+  .tablet({
+    text.primary, text.warning, text.primary-bg {
+      font-size: 0.8em;
+    }
+
+    table tbody tr td {
+      &:nth-child(4) {
+        span {
```

Comment:
I'd recommend the following 

```
{
  vertical-align: middle;
  font-size: 2em; 
}
```

The other statement's aren't necessary. The numbers align nicely with this change